scripts/kaggle/nodule_detection.py

```diff
@@ -44,7 +44,7 @@ def predict(model_id, model_weights, input_path, output_path):
     print('-' * 30)
     print('Predicting masks on test data...')
     print('-' * 30)
-    f_list = glob(input_path + "*.npy")
+    f_list = glob(input_path + os.pathsep + "*.npy")
```

Comment:
I thought it only works to construct concrete paths. Let me give a try and let you know.